### ISSUE TITLE
In order to unify syntax, standard JS should also have to invoke new to create StormDB instance

### ISSUE THREAD

**Issue by TomPrograms** [TomPrograms/stormdb#5]
I want to unify syntax so both Typescript and normal JS invoke `new` for the StormDB object like:

```js
const db = new StormDB(engine);
```

I have made the appropriate change to the code [here](https://github.com/TomPrograms/stormdb/commit/da22761aa408cffb61e777e207ca5ab2115d77ba). @geopic Would this require changes to `index.d.ts`? Thanks.

**Comment by geopic** [comment]
It would @TomPrograms. You want me to update the `index.d.ts` file?

**Comment by TomPrograms** [comment]
@geopic Yeah that would be great!

**Comment by geopic** [comment]
I'll get to work on it soon then.

**Comment by geopic** [comment]
@TomPrograms I've worked through on it, trying to get it to accept using the `new` keyword for both `StormDB` and the two static methods that return classes, and it hasn't come to fruition. :disappointed: Calling `new` doesn't work on static methods of a class. If you'd like for the developer to use `new` on all three classes then the only way is to export them all as individual properties of `module.exports` (instead of `StormDB` being the default export) so then the developer can import them like this:

```ts
import { StormDB, localFileEngine, browserEngine } from 'storm-db';
```

**Comment by TomPrograms** [comment]
Resolved by #6
